delta/app/src/main/resources/app.conf

```diff
@@ -183,6 +183,14 @@ app {
     pagination = ${app.defaults.pagination}
     # the projects indexing config
     indexing = ${app.defaults.indexing}
+
+    # configuration for the persistence of progress of projections
+    persist-progress-config {
+      # the maximum number of items in a batch before persisting progress
+      max-batch-size = 5
+      # the maximum time allowed to pass between persisting the progress
+      max-time-window = 50 millis
```

Comment:
the default values can be set to be higher... e.g. batch: 10, window: 5 seconds

Reply:
That can be problematic. The value is stored on the state **after** it is persisted on the DB. The higher this value the bigger the delay from the actual stream.

We could find ways to fix this (either branching the stream and doing some complicated broadcast & merge) or indeed using Distributed Data

Reply:
> The value is stored on the state after it is persisted on the DB.
The value should be added to the state for each element... and persisting to the DB should happen every now and again.

Reply:
I changed it. Now I'm using Distributed Data